net-libs/gloox/gloox-1.0.28-r1.ebuild

```diff
@@ -50,4 +50,9 @@ src_configure() {
 src_install() {
 	default
 	find "${ED}" -name "*.la" -delete || die
+
+	# unhide the libs directory
+	mv "${S}"/src/examples/.libs "${S}"/src/examples/libs
+
+	use examples && dodoc -r src/examples/
```

Comment:
I'd pair the "unhiding" operation with `if use examples ; then` since otherwise it's a useless operation. Not that it matters a lot.